srcpkgs/sparse/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'sparse'
+pkgname=sparse
+version=0.5.2
+revision=1
+build_style=gnu-makefile
+make_use_env=no
```

Comment:
The default is to not use environment -- by setting this string to anything you make it use environment.

Reply:
Just to elaborate a bit more: `common/build-style/gnu-makefile.sh` tests for

    if [ -z "$make_use_env" ]; then

So it only matters if it is empty or not. The actual value if set has no meaning.